packages/application-components/src/components/portals-container/portals-container.tsx

```diff
@@ -1,18 +1,66 @@
-import { css } from '@emotion/react';
+import { css, Global } from '@emotion/react';
 import { PORTALS_CONTAINER_ID } from '@commercetools-frontend/constants';
 
+type TPortalsContainerProps = {
+  /**
+   * The offset value for positioning the container from the top, when opened.
+   * Usually this is corresponds to the height of the header section.
+   */
+  offsetTop: string;
+  /**
+   * The CSS selector to apply the `overflow: hidden` style to (preventing scrolling)
+   * when a modal container is open.
+   */
+  containerSelectorToPreventScrollingOnOpen: string;
+  /**
+   * The `z-index` value to apply to the portal container. Default to `10000`.
+   */
+  zIndex: number;
+};
```

Comment:
These optional props allow to adjust things accordingly in case we use modal components outside of a Custom App.